static/js/place/chart_block.tsx

```diff
@@ -254,10 +255,11 @@ class ChartBlock extends React.Component<ChartBlockPropType> {
       let gotChart = false;
       if (
         !!this.props.data.isChoropleth &&
-        this.props.isUsaPlace &&
-        (this.props.placeType === "Country" ||
-          this.props.placeType === "State" ||
-          this.props.placeType === "County")
+        (isEarth ||
+          (this.props.isUsaPlace &&
+            (this.props.placeType === "Country" ||
```

Comment:
minor cleanup: I see that place.ts has a constant `placeTypesWithChoropleth` which is a set of all the possible place types to render a choropleth for.  Let's move that constant to the util.ts file in this same folder, rename it to USA_PLACE_TYPES_WITH_CHOROPLETH, export it and also use it here

Reply:
Done

Reply:
I see that you created the `USA_PLACE_TYPES_WITH_CHOROPLETH`, but didn't use it here?

Reply:
Pushed a new commit and modified the code